Automattic/wp-calypso
# Hosting Command Palette: Reset scroll position when new command list

Fixes https://github.com/Automattic/dotcom-forge/issues/4828

## Proposed Changes

Resets the scroll position when user navigates to a new command list (i.e. picking a command or going back)

https://github.com/Automattic/wp-calypso/assets/36432/7f7571a5-3044-44a2-a453-2be6f2d05aa0

## Testing Instructions

1. Open the command palette.
2. Scroll down a bit to a command that will open a list of sites.
3. Open the command.
4. Verify the scroll position for the site list has been reset.
